.github/workflows/terraform-lint.yaml

```diff
@@ -0,0 +1,25 @@
+name: Lint (Terraform)
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Hmmm. This PR puts the workflow on branch/v10 but tells it to run only on pushes to master.